lib/linguist/languages.yml

```
@@ -5488,6 +5488,49 @@ V:
   codemirror_mode: go
   codemirror_mime_type: text/x-go
   language_id: 603371597
+VB.NET:
+  type: programming
+  color: "#945db7"
+  extensions:
+  - ".vb"
+  - ".vbhtml"
+  aliases:
+  - visual basic
+  - visual basic .net
+  - vbnet
+  - vb .net
+  tm_scope: source.vbnet
+  ace_mode: text
+  codemirror_mode: vb
+  codemirror_mime_type: text/x-vb
+  language_id: 389
+VBA/VB6:
```

It's best not to have `/` in language names as this is very rare and can cause problems due to `/` being valid in *nix paths.

This is better named using the formal name: Visual Basic For Applications:

```suggestion
Visual Basic For Applications:
```

You'll need to adjust the samples' path too.

---

> This is better named using the formal name: Visual Basic For Applications.

Out of curiosity, is this the Linguist convention? Or something specific about VBA?

Also, it's a little hard to say which should be the primary name of the language -- it's the same language, but with different global objects, and different environments (VBA programs are stored within other file types, and run in the context of a host application; VB6 usually runs standalone). Nevertheless, if I had to choose, I would go with Visual Basic for Applications, as it hasn't really been replaced by anything else, and is still widely used; unlike VB6, for which there are a number of better alternatives.